usagov/usagov-2021
# Usagov 1219 home and close color

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1219

## Description

Change the home icon and text, as well as the close icon and text, of the mobile navigation to #1b1b1b.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


Open the mobile navigation menu and verify that both buttons and the text have the color #1b1b1b. Also, verify that the size of the icons matches the icons on usa.gov.
![image](https://github.com/usagov/usagov-2021/assets/39020809/91b1dbd2-4492-4387-b9e5-6a7067bb15c8)

### Requires New Config

### Requires New Content

### Validation Steps

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
